bindings/python/tests/test_output.py

```diff
@@ -1,11 +1,12 @@
-
 # Copyright 2023 IOTA Stiftung
 # SPDX-License-Identifier: Apache-2.0
 
 from iota_sdk import BasicOutput, AccountOutput, FoundryOutput, NftOutput, IssuerFeature, MetadataFeature
 from dacite import from_dict
+import pytest
 
 
+@pytest.mark.skip(reason="temporarily skipped until 2.0 blocks in use")
```

Comment:
I don't get it, what is preventing this test from passing?

Reply:
nothing, I was too eager here.

Reply:
Included again.